index.html

```diff
@@ -432,6 +432,75 @@ <h3>Obtaining Consent</h3>
         </li>
       </ol>
     </section>
+    <section>
+      <h3 id=permissions-integration>Permissions Integration</h3>
+      <p>The Audio Output Devices API is a [=powerful feature=] that is
+      identified by the [=powerful feature/name=] "speaker-selection".</p>
+      <p>It defines the following types and algorithms:</p>
+      <dl>
+        <dt>
+          [=powerful feature/permission descriptor type=]
+        </dt>
+        <dd>
+          <p>
+            A permission covers access to the device given in the associated
+            {{DevicePermissionDescriptor}} descriptor.
+          </p>
+          <p>
+            If the descriptor does not have a {{DevicePermissionDescriptor/deviceId}}, its
+            semantic is that it queries for access to all devices of that class. Thus, if a query
+            for the "speaker-selection" [=powerful feature=] with no
+            {{DevicePermissionDescriptor/deviceId}} returns {{PermissionState/"granted"}}, the
+            client knows that there will never be a permission prompt for an audio output device,
+            and if {{PermissionState/"denied"}} is returned, it knows that no getUserMedia request
+            for an audio output device will succeed.
+          </p>
+          <p>
+            If a permission state is present for access to some, but not all, audio output devices,
+            a query without the {{DevicePermissionDescriptor/deviceId}} will return
+            {{PermissionState/"prompt"}}.
```

Comment:
I wasn't aware of this. What normative text ensures this? Do you have a link?

Reply:
@miketaylr Thanks for digging that up. I asked @alvestrand about it and am OK with merging this.